lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -85,19 +74,16 @@ describe('SimpleOfflineCache instance', function () {
     });
   });
 
-  it('should fetch from network and cache if the resource is not available',
+  it('should fetch from network if the resource is not available',
   function (done) {
     var offline = new SimpleOfflineCache('test');
     var request = new Request('/');
+    sinon.spy(self, 'fetch');
     offline.onFetch(request.clone(), null)
     .then(function () {
-      return caches.open('test');
-    })
-    .then(function (cache) {
-      return cache.match(request.clone());
-    })
-    .then(function (response) {
-      expect(response).to.be.ok;
+      expect(fetch.calledOnce).to.be.true;
+      expect(fetch.getCall(0).args[0].url).to.equal(request.url);
+      fetch.restore();
```

Comment:
You should add this in the `catch()` handler as well to allow other tests to continue their execution independently.